### PR TITLE
[W5.11][W10-A4]Jackson Huang

### PR DESCRIPTION
**Pull request by jaxony**
Implemented case-insensitive `find` as enhancement 2

### CODE REVIEW

**Comment by Steven-Jia** [comment]
Looks good to me!

File: src/seedu/addressbook/data/person/Name.java [hunk 1]
```diff
@@ -42,6 +43,18 @@ public static boolean isValidName(String test) {
     public List<String> getWordsInName() {
         return Arrays.asList(fullName.split("\\s+"));
     }
+    
+    /**
+     * Retrieves a listing of every word in the name, in order, in lowercase.
+     */
+    public List<String> getWordsInNameLowerCase() {
```
Comment (tshradheya) [inline src/seedu/addressbook/data/person/Name.java:50]: Better naming is `getWordsInNameInLowerCase()`

File: test/java/seedu/addressbook/commands/FindCommandTest.java [hunk 2]
```diff
@@ -25,8 +25,8 @@ public void execute() throws IllegalValueException {
         //same word, same case: matched
         assertFindCommandBehavior(new String[]{"Amy"}, Arrays.asList(td.amy));
 
-        //same word, different case: not matched
-        assertFindCommandBehavior(new String[]{"aMy"}, Collections.emptyList());
+        //same word, different case: matched
+        assertFindCommandBehavior(new String[]{"aMy"}, Arrays.asList(td.amy));
```
Comment (tshradheya) [inline test/java/seedu/addressbook/commands/FindCommandTest.java:29]: Good job updating JUnit Tests

**Review by tshradheya — COMMENTED** [review]
Good work with the enhancement 🎉

**Pull request closed without merging**